lib/dsp.js

```diff
@@ -0,0 +1,2301 @@
+/* 
```

Comment:
Actually, since the DSP library should only be used in the example, to showcase how it could be used with the oscillator, you should add it as a devDependency